### PR TITLE
Gauge charts

### PR DESCRIPTION
**Pull request by teamconnect**

### CODE REVIEW

File: examples/examples.html [hunk 1]
```diff
@@ -4,6 +4,7 @@
     <link href='http://fonts.googleapis.com/css?family=Open+Sans' rel='stylesheet' type='text/css'>
     <link href='http://fonts.googleapis.com/css?family=Roboto:400,700' rel='stylesheet' type='text/css'>
     <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/c3/0.4.10/c3.css" />
+    <link rel="stylesheet" type="text/css" href="http://code.ionicframework.com/ionicons/2.0.1/css/ionicons.min.css" />
```
Comment (AdamPD) [inline examples/examples.html:7]: Where is this used?

File: lib/viz/chart/chart.ts [hunk 2]
```diff
@@ -28,25 +35,42 @@ class Chart implements Common.Visualization {
...
+            firstSelect = metadata.selects[0],
+            options = this._options,
+            fieldOptions = options.fieldOptions,
+            type = options.type;
+
+        _.each(fields, (fieldName) => {
+            fieldOptions[fieldName] = fieldOptions[fieldName] || {}
+        });
+
+        if (type === 'gauge'){
```
Comment (AdamPD) [inline lib/viz/chart/chart.ts]: This seems fairly specific and "boundary breaking" - what's the need for it?

File: examples/examples.html (hunk 1 above)
Comment (teamconnect) [inline examples/examples.html:7]: In the errors

**Comment by AdamPD** [comment]
+1

**Pull request merged**